commcare_connect/templates/account/base.tailwind.html

```diff
@@ -0,0 +1,57 @@
+{% load static i18n %}
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>{% block title %}CommCareConnect - {% block head_title %}{% endblock head_title %}{% endblock title %}</title>
```

Comment:
nit: should have space in the name

Reply:
Fixed. [ba68d48](https://github.com/dimagi/commcare-connect/pull/581/commits/ba68d48e47956a171ded90486668d1b51ebcb73c)